event/src/main/java/net/md_5/bungee/event/EventBus.java.orig

```diff
@@ -0,0 +1,245 @@
+package net.md_5.bungee.event;
```

Comment:
Is this file supposed to still exist?
I was in the middle of making a pr downstream at waterfall to swich to LambdaMatafactory.
Really looking forward to this change. 
Good work!

Reply:
This is a leftover file from me merging, i'll remove it